.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python: 3.7
+sudo: true
+dist: bionic
```

Comment:
We should agree on this as well to have a default across all the projects. \cc @lwsanty

Reply:
I have chosen the most up-to-date environment here.

Reply:
we don't want to run ubuntu (desktop distro) on the server, right?
So maybe it makes sense to have the same distro on travis and in Docker file.

Reply:
Python does not play well with any musl-like. It is very hard to install packages, and those that install sometimes don't function well. Ubuntu is a golden standard here.

Regarding `ubuntu (desktop distro)`: c'mon, an Ubuntu docker image is `apt-get` with `coreutils`, there is nothing else.
It's called "core-cloud" nowadays, https://partner-images.canonical.com/core/bionic/current/
As long as you don't `apt-get install` much, you don't install much of bloated crap.

Reply:
let's use the latest(afaik in travis it is bionic) across all the projects, the dafault one which is xenial seems to be too outdated

please correct me if I'm wrong

Reply:
Yep. Proof: https://docs.travis-ci.com/user/reference/overview/#linux

Reply:
Yup, agree with @lwsanty